server/src/child.rs

```
@@ -251,10 +256,14 @@ fn run(
                 }
             }
 
-            ServerMessage::Stop => {}
+            ServerMessage::Stop => {} // how to stop a long running trace
 
             ServerMessage::Quit => {
-                let _ = target.child.terminate();
```

This should still ignore the error- if termination fails there's nothing we can do about it and we're just going to abandon it anyway.